src/api/tests/feature_tests/features/createDeviceMessageHandlingSystem.success.feature

```diff
@@ -15,12 +15,19 @@ Feature: Create MHS Device - success scenarios
       | keys.0.key_type  | product_team_id_alias |
       | keys.0.key_value | FOOBAR                |
     And I note the response field "$.id" as "product_team_id"
-    And I have already made a "POST" request with "default" headers to "ProductTeam/${ note(product_team_id) }/Product/Epr" with body:
+    And I have already made a "POST" request with "default" headers to "ProductTeam/<product_team_id>/Product/Epr" with body:
```

Comment:
Does <product_team_id> work?

Reply:
if you use a `Scenario Outline` rather than `Scenario` then `<product_team_id>` is the table header of one the `Examples`